fetch_gazebo/robots/fetch.gazebo.xacro

```diff
@@ -1,6 +1,50 @@
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="fetch" >
   <xacro:include filename="$(find fetch_description)/robots/fetch.urdf" />
 
+  <!-- Add four casters to the base -->
+  <xacro:macro name="caster" params="prefix joint_x joint_y">
+  <link name="${prefix}_caster_link">
+    <visual>
+      <origin rpy="0 0 0" xyz="0 0 0"/>
+      <geometry>
+        <sphere radius="0.0601"/> <!-- 0.06033 -->
```

Comment:
The value in comments, 0.06033 is the actual value of the wheel, but I've made the caster slightly smaller.

When the values were all the same as the wheel, the wheels did not have enough contact with the ground.